schema/rdl_schema_0.1.json

```
@@ -151,17 +151,56 @@
               "required": [
+                "id",
                 "hazard_type",
                 "analysis_type"
               ],
               "properties": {
+                "id": {
+                  "title": "Event set identifier",
+                  "type": "string",
+                  "description": "The unique identifier for this event set.",
+                  "minLength": 1
+                },
                 "hazard_type": {
-                  "$ref": "#/$defs/common_hazard_type",
-                  "description": "Type of hazard modelled"
+                  "title": "Hazard type",
+                  "type": "string",
+                  "description": "The main type of hazard phenomena in the modelled scenario(s).",
+                  "codelist": "hazard_type.csv",
+                  "openCodelist": false
                 },
                 "analysis_type": {
-                  "$ref": "#/$defs/common_analysis_type",
-                  "description": "Type of analysis, probabilistic or deterministic"
+                  "title": "Analysis type",
+                  "type": "string",
+                  "description": "The type of analysis used by the hazard model: probabilistic (multiple potential events characterised by different intensity and occurrence frequency); deterministic (frequency-independent representation, such as mean, median or max); empirical (obtained from real event observations).",
+                  "codelist": "analysis_type.csv",
+                  "openCodelist": false
+                },
+                "frequency_distribution": {
+                  "title": "Frequency distribution",
+                  "type": "string",
+                  "description": "The frequency distribution assumed for the occurrence of events over a multi-year timeline.",
+                  "codelist": "frequency_distribution.csv",
+                  "openCodelist": false
+                },
+                "seasonality": {
+                  "title": "Seasonality distribution",
+                  "type": "string",
+                  "description": "The seasonality distribution assumed for the occurrence of events across a calendar year.",
+                  "codelist": "seasonality.csv",
+                  "openCodelist": false
+                },
+                "calculation_method": {
+                  "title": "Calculation Method",
+                  "type": "string",
+                  "description": "The methodology used for the calculation of the event in the modelled scenario(s).",
+                  "codelist": "data_calculation_type.csv",
+                  "openCodelist": false
+                },
+                "number_events": {
+                  "title": "Number of events",
+                  "type": "number",
+                  "description": "The number of events contained in the event set. You should only use this field when details of individual events are not included in the RDLS metadata."
```

In keeping with other field names and titles which use a `concept_attribute` formulation.

```suggestion
                "event_count": {
                  "title": "Event count",
                  "type": "number",
                  "description": "The number of events contained in the event set. You should only use this field when details of individual events are not included in the RDLS metadata."
```